flight/states/early_laps.py

```diff
@@ -16,106 +16,26 @@ async def arange(count):
     for i in range(count):
         yield i
 
-
 class EarlyLaps:
     """Handles getting the drone around the two pylons 8 times"""
 
     async def run(self, drone):
         """Moves the drone to the first pylon, then begins the 8 laps"""
         # Go to pylon 1
         logging.info("Moving to pylon 1")
-        await self.wait_pos(drone, config.pylon1)
+        await D_Move.moveTo(self, drone, config.pylon1)
         logging.info("Arrived at pylon 1")
         async for i in arange(config.NUM_LAPS):
             logging.info("Starting lap: %d", i)
             logging.debug("Lap %d: Straight one", i)
-            await self.wait_pos(drone, config.pylon2)  # move to pylon 2
+            await D_Move.moveTo(self, drone, config.pylon2)  # move to pylon 2
 
             logging.debug("Lap %d: Turn one", i)
-            await self.wait_turn(drone)  # turn around pylon 2
+            await D_Move.turnTo(self, drone)  # turn around pylon 2
 
             logging.debug("Lap %d: Straight two", i)
-            await self.wait_pos(drone, config.pylon1)  # move to pylon 1
+            await D_Move.moveTo(self, drone, config.pylon1)  # move to pylon 1
 
             logging.debug("Lap %d: Turn two", i)
-            await self.wait_turn(drone)  # turn around pylon 1
+            await D_Move.turnTo(self, drone)  # turn around pylon 1
```

Comment:
It doesn't make much sense to pass `self` into these functions, since `self` is of type `EarlyLaps` and is not used within the `moveTo` and `turnTo` functions. Currently these functions are being called on the class itself, rather than an instance of the class. To resolve this, all we have to do is instantiate an object before calling these functions:

```
mover: D_Move = D_Move()

class EarlyLaps:
...
```

Which then allows us to call the functions on the object:
`await mover.moveTo(drone, config.pylon1)`
`await mover.turnTo(drone)`
without needing to pass in an extraneous `self` object.

Reply:
Here's an example that's based on the naming changes suggested in this review:
```
movement_controller: MovementController = MovementController()

class EarlyLaps:
...

await movement_controller.move_to(drone, config.pylon1)
await movement_controller.turn(drone)
```